src/pages/myPage/components/Tabs/Tabs.style.ts

```diff
@@ -0,0 +1,22 @@
+import { Theme, css } from '@emotion/react';
+import { flexGenerator } from '@styles/generator';
+
+export const userDistinguishWrapper = css`
+  ${flexGenerator('row', 'flex-start', 'flex-start')};
+  padding-left: 2rem;
```

Comment:
```suggestion
  padding: 1.4rem 2rem 3rem 2rem;
```

이런식으로 padding-left만 주는것보다 해당 부분을 하나의 section으로 보면서 구조를 짜면 더 좋을것 같습니다. 굳이 지금 수정은 안해도 될 것 같아요!

Reply:
아 tabs 컴포넌트를 공용으로 사용해볼까 했다가 각각 페이지에서 따로 구현하는 방식으로 바뀌어서 이 Tabs컴포넌트는 필요가 없어졌습니다 ㅠ 이 파일 삭제할게요!